js/notify/electron-notify.js

```diff
@@ -422,6 +423,14 @@ function setNotificationContents(notfWindow, notfObj) {
     // Display time per notification basis.
     let displayTime = notfObj.displayTime ? notfObj.displayTime : config.displayTime;
 
+    getConfigField('alwaysOnTop').then((isAlwaysOnTop) => {
+        if (isAlwaysOnTop) {
```

Comment:
this is asynchronous and by the time we get the data from config, this process would've finished and it won't work.

please have a local variable for alwaysOnTop, read that during the initialisation process, update it when the setting changes and use that variable to make this check.